package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "css-slam",
-  "version": "1.2.0",
+  "version": "1.2.1",
```

Comment:
Don't update the version number in a pull request, we'll do that when we stamp a new release based on any other unreleased changes.